src/Promitor.Core.Scraping/Configuration/Model/MetricsDeclaration.cs

```diff
@@ -5,6 +5,7 @@ namespace Promitor.Core.Scraping.Configuration.Model
 {
     public class MetricsDeclaration
     {
+        public bool UseAzureMonitorSdk { get; set; }
```

Comment:
Let's migrate by default.

```suggestion
        public bool UseAzureMonitorSdk { get; set; } = true;
```